org.eclipse.xtext.util/pom.xml

```diff
@@ -15,7 +15,10 @@
 		<dependency>
 			<groupId>org.eclipse.xtend</groupId>
 			<artifactId>org.eclipse.xtend.lib</artifactId>
-			<!-- TODO: update it -->
+			<!--
+			TODO: update it
+			The dev BOM should be in the same Git repository of this one
+			-->
 			<version>${xtext-current-release-version}</version>
```

Comment:
shouldnt this be project.version?

Reply:
I'm writing the notes for this PR (including this part), but since you asked: this is feasible (easily) only if the xtext-lib projects are in the same Git repository (which I'd like to do as well); otherwise, we could still do that now if we enable the snapshot repository pointing to the latest Jenkins build. (note that currently the Maven repository in the POM are disabled).

Reply:
@LorenzoBettini we still can do it,
as we do here
https://github.com/eclipse/xtext-maven/blob/be49c0cf901f9110e57ff3342806b6ef0878d8bd/org.eclipse.xtext.maven.parent/pom.xml#L22
https://github.com/eclipse/xtext-maven/blob/be49c0cf901f9110e57ff3342806b6ef0878d8bd/org.eclipse.xtext.maven.parent/pom.xml#L285

or here
https://github.com/eclipse/xtext-core/blob/62b0ee64f3590befaabc1066fca185e0bec32dba/gradle/upstream-repositories.gradle#L30

Reply:
OK, I'll try to re-enable that.

Reply:
This https://github.com/eclipse/xtext-core/pull/2054/commits/65f209b8838c9f56d30e02a14701cf706a96f1a6 should implement it; for the moment, it's only useful for xtext-lib; xtend-maven-plugin is still taken from the latest release or milestone. Of course, it could be adapted as well.

Reply:
for xtend maven plugin we eplicitely want to use a bootstraped older version

Reply:
OK, so that's fine as it is now, isn't it? Or do you prefer a released version (2.29.0)?

Reply:
no we want to bootstrap against M1